kyuubi-common/src/main/resources/log4j2-defaults.properties

```diff
@@ -29,7 +29,7 @@ appender.console.layout.pattern = %d{HH:mm:ss.SSS} %p %c: %m%n
 appender.console.filter.1.type = Filters
 
 appender.console.filter.1.a.type = ThresholdFilter
-appender.console.filter.1.a.level = fatal
+appender.console.filter.1.a.level = INFO
```

Comment:
Minor: should we keep in lower case to align with others?

Reply:
Agree to @link3280 . If need change, should better modify global.